traffic.sh

```diff
@@ -3,41 +3,180 @@
 # Licensed under the BSD 3-Clause License.
 # SPDX-License-Identifier: BSD-3-Clause
 
```

Comment:
can you explain a bit more what problem is being solved with this PR?

Reply:
if you specify a prefix in the clab.yml then the traffic.sh tries to communicate with nodes by the wrong name.   Just trying to make the script a bit more portable, and allow for quicker reuse in other labs.